### PR TITLE
ADD PMM-T1328 - Verify public address is set automatically

### PR DESCRIPTION
**Pull request by VoK0**

### CODE REVIEW

**Comment by puneet0191** [comment]
@VoK0 execution results?

**Comment by VoK0** [comment]
This needs to merged first otherwise the job will always fail because of the missing env variables and therefore API call to portal is not possible 
Here is POC on my tmp job which is just clone of pmm-ui-nightly with fixed variables https://pmm.cd.percona.com/job/VokoTmpNightlyUITests/11/allure/#suites/d9805975708c0a76781ecd3ab76468a5/3a6367079ef419c8/

tmp Job link https://pmm.cd.percona.com/blue/organizations/jenkins/VokoTmpNightlyUITests/detail/VokoTmpNightlyUITests/11/pipeline

Failed tests are unrelated to my changes

File: tests/pages/perconaPlatformPage/perconaPlatformPage.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ module.exports = {
     requiredField: 'Required field',
     invalidEmail: 'Invalid email address',
     connectedSuccess: 'Successfully connected PMM to Percona Platform',
+    UpdateSuccess: 'Settings updated',
```
Comment (peterSirotnak) [inline tests/pages/perconaPlatformPage/perconaPlatformPage.js:39]: please use camelCase in variable name

File: tests/pages/perconaPlatformPage/perconaPlatformPage.js [hunk 2]
```diff
@@ -74,10 +75,13 @@ module.exports = {
     I.seeTextEquals('', this.elements.emailValidation);
   },
 
-  connectToPortal(token, serverName = 'Test Server') {
+  connectToPortal(token, serverName = 'Test Server', connectedViaPortalApi = false) {
```
Comment (peterSirotnak) [inline tests/pages/perconaPlatformPage/perconaPlatformPage.js:78]: connectedViaPortalApi, change please to something that describes it better, probably isPublicAddressSet

File: tests/configuration/verifyPMMSettingsPageFunctionality_test.js [hunk 3]
```diff
@@ -347,6 +347,32 @@ Scenario(
...
+    await pmmSettingsPage.verifyTooltip(pmmSettingsPage.tooltips.advancedSettings.publicAddress);
+
+    await I.waitForVisible(pmmSettingsPage.fields.publicAddressInput, 30);
+    I.seeElement(pmmSettingsPage.fields.publicAddressButton);
+   
+    const publicAddressValue = await I.grabValueFrom(pmmSettingsPage.fields.publicAddressInput);
+    const serverAddressIP = process.env.VM_IP;
+
+    I.assertTrue(publicAddressValue.length > 0, 'Expected the Public Address Input Field to be not empty!');
+    I.refreshPage();
```
Comment (peterSirotnak) [inline tests/configuration/verifyPMMSettingsPageFunctionality_test.js:369]: Is it really necessary to refresh page? Is public IP not present?

**Review by peterSirotnak — APPROVED** [review]

**Pull request merged**